zfsbootmenu/install-helpers.sh

```diff
@@ -100,6 +100,12 @@ create_zbm_conf() {
     has_column=1
   fi
 
+  local busybox_has_dmesg
+  dmesg="$( find "${BUILDROOT}" -name dmesg )"
```

Comment:
This feels a bit hackish, but I don't want to assume it's always at `${BUILDROOT}/usr/bin/dmesg`, and we can't use `$PATH` right here.

Reply:
I also don't want to do a generic "has busybox" test, because the modularity of busybox means that it might not ship dmesg.